src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,114 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import java.util.ArrayList;
+
+
```

Comment:
Based on conventions [here](https://se-education.org/guides/conventions/java/intermediate.html), it might be better to remove the extra empty lines here (perhaps only one empty line is needed). : )